### ISSUE TITLE
Bane Shield not triggering properly, doing wrong damage

### ISSUE THREAD

**Issue by Skullgrin** [WarEmu/WarBugs#20557]
As can be seen in the following combat log screen shots, Bane Shield is not triggering each time I am being hit. 

Here is the Bane Shield tooltip which displays the expected behavior -  150 damage to each person that hits you, for 10 seconds. no limit on targets or cooldown between each activation is listed either.

![Bane Shield](https://user-images.githubusercontent.com/40448982/202607822-6b7009a9-223c-4ddb-ab76-d0713ecfe6b3.jpg)

In the following screen shots you can see its actual behavior -

![baneshield2](https://user-images.githubusercontent.com/40448982/202608939-dce23e2f-a1fa-4215-81be-d81d3e854a72.jpg)

In this first picture, Bane Shield first activates at 19:38:34 but only hits one of the 5 people attacking me. Also of note is the fact that it only does 99 damage rather than the 150 damage the tooltip states. The next time it activates is at 19:38:36, hitting a different target attacking me this time, however I have been hit 6 times by others and it did not activate for them during this 2 second period.

![baneshield3](https://user-images.githubusercontent.com/40448982/202608942-e92b07ec-e09d-4074-8124-c5aee3fe14de.jpg)

Bane Shield next triggers one second later at 19:38:37. This time I have received 10 hits for which it did not activate.
  
![baneshield4](https://user-images.githubusercontent.com/40448982/202608947-5382c632-bb5f-45f0-88e7-c6b2ae6da6fd.jpg)

In this final picture Bane Shield triggers at 19:38:39, 19:38:40, and 19:38:42. Again activating fewer times than I was hit.

Looking closer at these activations it also seems to be prioritizing targeting the White Lions pet rather than the player himself, which may be yet another bug with this ability.

 I do acknowledge some of the attacks above could be residual DoT's, but even so all of the players attacking me should have received at least one damaging attack from Bane Shield, and for several of them it should have triggered multiple times over the ten second period it was up. 

In conclusion, Bane Shield seems to be massively broken. It is doing less damage than the tooltip states, not activating properly (it appears to have some sort of internal cooldown that is interfering with it triggering correctly on your being hit) and also seems to be mis-targeting in the case of pets. It should work similarly to the way that damaging auras work, both of which can be seen working properly in the above screen shots.

**Comment by Zomega-WAR** [comment]
Bane Shield has a 1s ICD as per: https://returnofreckoning.com/forum/viewtopic.php?t=37646&start=20#p407458

**Comment by Zomega-WAR** [comment]
Also, do you have a screenshot of your own Bane Shield tooltip in-game rather than from the career builder? How many points did you have in Path of Corruption (I'm not sure if the damage scales based on mastery points).
